examples/Workflow/WorkflowConsoleApp/Activities/UpdateInventoryActivity.cs

```diff
@@ -39,7 +39,7 @@ public override async Task<object> RunAsync(WorkflowActivityContext context, Pay
                 this.logger.LogInformation(
                     "Payment for request ID '{requestId}' could not be processed. Insufficient inventory.",
                     req.RequestId);
-                throw new InvalidOperationException();
+                throw new InvalidOperationException("Not enough inventory!");
```

Comment:
Not a blocker for this review, but can we include the item that we are trying to purchase here as well as the number of items still available of that type for purchase?

Reply:
Agreed, that would have been better. It's actually really difficult to hit this case today since the ReserveInventoryActivity will do this check already before we get here, so I don't think users playing with the sample will ever hit this, but I'll update it if there are other blocking things that need to be done for this PR.